front/src/components/LoginPanel/comp/RegisterForm.test.js

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from "@testing-library/react";
+import { MockedProvider } from "@apollo/client/testing";
+import RegisterForm from "./RegisterForm";
+
+describe("RegisterForm", () => {
+    beforeEach(() => {
+        render(
+            <MockedProvider mocks={[]} addTypename={false}>
+                <RegisterForm />
+            </MockedProvider>
+        );
+    });
+
+    it("should have three textboxes", () => {
+        let textInputs = screen.getAllByRole("textbox");
+        expect(textInputs).toHaveLength(3);
+    });
```

Comment:
Consider writing 3 short tests for each of the textboxes. I think that getByLabelText will be the best choice, but getByRole may not be worse.
Using `/pattern/i' allows to do case-insensitive search by substring. Use the core, most-important part of text, eq. "zarejestuj" not "zarejestruj się", so tests will still have some flexibility and will not break after cosmetic changes.